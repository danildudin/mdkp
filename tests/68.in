24 6
35 353 713 40 921 534 602 485 43 835 519 945 289 398 536 121 956 297 749 223 599 643 746 830 
517 116 448 381 48 216 831 119 350 674 378 789 258 185 549 426 68 414 812 13 781 908 333 573 4964
961 502 505 131 170 829 493 398 425 328 520 31 785 427 605 781 346 581 503 1 803 943 51 342 2774
323 129 626 564 961 748 764 401 576 474 319 410 227 819 206 486 708 149 132 142 445 715 313 844 2990
119 831 480 539 21 967 746 487 435 669 609 256 992 26 280 456 309 846 853 656 614 432 402 712 1626
734 146 335 442 210 783 217 775 150 679 964 711 77 112 801 613 653 990 100 23 438 530 786 256 1357
939 232 112 624 983 704 868 996 35 173 776 325 887 39 930 576 999 574 694 781 443 608 532 46 1272
