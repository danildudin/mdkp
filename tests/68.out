2805
4
2 11 13 18 
